src/mono/sample/wasm/wasm.mk

```diff
@@ -22,15 +31,15 @@ clean:
 	rm -rf bin $(TOP)/artifacts/obj/mono/$(PROJECT_NAME:%.csproj=%)
 
 run-browser:
-	if ! $(DOTNET) tool list --global | grep dotnet-serve; then \
+	if ! $(DOTNET) tool list --global | grep dotnet-serve && ! which dotnet-serve ; then \
 		echo "The tool dotnet-serve could not be found. Install with: $(DOTNET) tool install --global dotnet-serve"; \
```

Comment:
you can add dotnet-serve into https://github.com/dotnet/runtime/blob/main/.config/dotnet-tools.json then it will be automatically installed as part of the repo dotnet restore and available via `$(DOTNET) serve`.

Reply:
We'll do it in a follow-up PR; concurrent with bumping `dotnet-public` to include a newer `dotnet-serve` (it's got 1.8.15, while nuget.org is on 1.10.82)

Reply:
But do we want every dotnet/runtime developer (and CI) to pay the price of restoring that tool?

Reply:
The version changes very infrequently so locally you'll get it from the nuget cache and I don't think it matters for CI (we already have e.g. reportgenerator or coverlet which is not used by many)